### DISCUSSION TITLE
Any benefit of using native-only components over universal components?

### DISCUSSION THREAD

**Discussion by shovel-kun** [founded-labs/react-native-reusables#223] [Q&A]
Hi.

According to this old comment (https://github.com/mrzachnugent/react-native-reusables/issues/27#issuecomment-1914742913), it seems like you're gradually making native components more web-friendly. Currently, there are two ways of using `rn-reusables`: native-only or universal.

1. Do you plan on continually supporting native-only components in the future?
2. Is there any benefit in choosing native-only over universal components in a mobile-only project?

Thanks!

**Comment by mrzachnugent** [comment]
Good question.

This is tech-debt. Once upon a time, the `@rn-primitives` were not npm packages. The files were added into your `/components/primitives` folder and depending on the selection of "native-only" or "universal", it would add different files into your primitives folder.

Added an issue to fix this: https://github.com/mrzachnugent/react-native-reusables/issues/228
